.github/workflows/ci_e2e.yaml

```diff
@@ -0,0 +1,77 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: Tests
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  schedule:
+    - cron: '53 0 * * SAT' # Every Saturday at 00:53 UTC
```

Comment:
**thought:** I might add `on: workflow_dispatch` here, or some equivalent.

Maybe not needed now-now, but I'm wondering if there's a world where we trigger a workflow run from `X-operator` on PR, and use that as part of the CI there too. I wonder if that's possible?

Reply:
I don't know to be honest, I need to investigate about it.